OpTestConfiguration.py

```diff
@@ -87,9 +87,9 @@ def get_parser():
                           help="[QEMU Only] qemu simulator binary")
 
     hostgroup = parser.add_argument_group('Host', 'Installed OS information')
-    hostgroup.add_argument("--host-ip", help="Host address")
-    hostgroup.add_argument("--host-user", help="SSH username for Host")
-    hostgroup.add_argument("--host-password", help="SSH password for Host")
+    hostgroup.add_argument("--host-ip", help="Host address", required=True)
```

Comment:
@PraveenPenguin It is better to add these checks in testcase as how we do it for host-mac, host-ip etc...
it may be dependant for all the testcases.

Reply:
@sathnaga thanks for review , yes make sense will do